airbyte-cdk/java/airbyte-cdk/db-destinations/src/testFixtures/kotlin/io/airbyte/cdk/integrations/standardtest/destination/typing_deduping/JdbcSqlGeneratorIntegrationTest.kt

```diff
@@ -146,8 +146,10 @@ abstract class JdbcSqlGeneratorIntegrationTest<DestinationState : MinimumDestina
         includeCdcDeletedAt: Boolean,
         streamId: StreamId,
         suffix: String?,
-        records: List<JsonNode>
+        records: List<JsonNode>,
+        generationId: Long,
     ) {
+        // TODO handle generation ID
```

Comment:
@gisripa I'm leaving the jdbc sqlgenerator integration test unimplemented for now. It's probably (?) pattern-matchable from https://github.com/airbytehq/airbyte/pull/38359/files#diff-39dbfe0f01e0b6a3efd7e11401344874227b2e30e04229391b38cbb9893913b2

Reply:
ack. Seems similar. for now its just no-op for old jdbc connectors right ?

Reply:
yep. in theory this means they'll continue to build+test successfully, though... I haven't validated that in any way